command-expansion-server/sql/commanding/tables/command_dictionary.sql

```diff
@@ -25,3 +26,17 @@ comment on column command_dictionary.version is e''
   'A human-meaningful version qualifier.';
 comment on constraint command_dictionary_natural_key on command_dictionary is e''
   'There an only be one command dictionary of a given version for a given mission.';
+
+CREATE FUNCTION trigger_update_at_timestamp()
```

Comment:
Can we use lowercase for SQL keywords to stay consistent with the other files?

Reply:
Sure, no problem